[2025-10-13T19:30:41.468+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scanfile.get_webdriver_options_a scheduled__2025-10-12T00:00:00+00:00 [queued]>
[2025-10-13T19:30:41.560+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scanfile.get_webdriver_options_a scheduled__2025-10-12T00:00:00+00:00 [queued]>
[2025-10-13T19:30:41.574+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-10-13T19:30:41.653+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): get_webdriver_options_a> on 2025-10-12 00:00:00+00:00
[2025-10-13T19:30:41.752+0000] {standard_task_runner.py:57} INFO - Started process 84 to run task
[2025-10-13T19:30:41.902+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'scanfile', 'get_webdriver_options_a', 'scheduled__2025-10-12T00:00:00+00:00', '--job-id', '66', '--raw', '--subdir', 'DAGS_FOLDER/scanfile.py', '--cfg-path', '/tmp/tmpdmtdsqnj']
[2025-10-13T19:30:41.974+0000] {standard_task_runner.py:85} INFO - Job 66: Subtask get_webdriver_options_a
[2025-10-13T19:30:42.734+0000] {task_command.py:416} INFO - Running <TaskInstance: scanfile.get_webdriver_options_a scheduled__2025-10-12T00:00:00+00:00 [running]> on host afd0d3e13103
[2025-10-13T19:30:43.434+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='scanfile' AIRFLOW_CTX_TASK_ID='get_webdriver_options_a' AIRFLOW_CTX_EXECUTION_DATE='2025-10-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-12T00:00:00+00:00'
[2025-10-13T19:30:43.465+0000] {python.py:194} INFO - Done. Returned value was: hello world
[2025-10-13T19:30:43.625+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=scanfile, task_id=get_webdriver_options_a, execution_date=20251012T000000, start_date=20251013T193041, end_date=20251013T193043
[2025-10-13T19:30:43.751+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-10-13T19:30:43.972+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
